README.md

```diff
@@ -39,3 +39,39 @@ To upload a dashboard while editing, run:
 * grafonnet: https://github.com/grafana/grafonnet-lib
 
 `git clone https://github.com/grafana/grafonnet-lib.git $GOPATH/src/github.com/grafana/grafonnet-lib`
+
+
+
+### Mixin
```

Comment:
@TheoBrigitte Here is the note about the update.

Reply:
Perfect that's very clear, thanks <3